hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/DNCertificateClient.java

```diff
@@ -25,20 +25,26 @@
 import org.slf4j.LoggerFactory;
 
 import org.apache.hadoop.hdds.security.x509.SecurityConfig;
+
+import java.nio.file.Paths;
+
 /**
  * Certificate client for DataNodes.
  */
 public class DNCertificateClient extends DefaultCertificateClient {
 
   private static final Logger LOG =
       LoggerFactory.getLogger(DNCertificateClient.class);
+
+  public static final String COMPONENT_NAME = Paths.get("dn").toString();
```

Comment:
Minor NIT: Here we can use directly assign the component name, instead of calling Paths.get("dn").toString(). As this also just returns "dn".